pkg/logs/internal/status/info.go

```diff
@@ -86,15 +82,15 @@ func NewMappedInfo(key string) *MappedInfo {
 
 // SetMessage sets a message with a unique key
 func (m *MappedInfo) SetMessage(key string, message string) {
-	defer m.lock.Unlock()
-	m.lock.Lock()
+	defer m.mu.Unlock()
+	m.mu.Lock()
```

Comment:
I know it doesn't change anything but it seems weird to me to defer `Unlock` before even calling `Lock`, do you know if there is any reason it was written like that ?

Reply:
No good reason. I re-arranged it.